.env

```diff
@@ -3,12 +3,9 @@ PORT=8080
 PGHOST=db
 PGPORT=5432
 PGUSER=postgres
-PGPASSWORD=password
```

Comment:
@ZeRego - did you mean to change this file 🤔 not sure if this is related to the install docs

Reply:
I'm removing the default values for important fields from the docker compose. 
This will remove the risk of people deploying to production with default values and be easily hacked. 
This way we force people to setup their own database password and Lightdash secret.

These default values will still be set when using the install script for a quick install.

cc: @owlas

Reply:
One user actually requested that we move this to `.env.template` that way it won't be used by default

Reply:
I think the rest of the defaults are fine and we now have clear docs showing what vars they can set and what are their default values.